website/docs/docs/build/incremental-models.md

```diff
@@ -303,5 +303,64 @@ select ...
 
 </File>
 
+<VersionBlock firstVersiont="1.4">
+
+### About incremental_predicates
+
+`incremental_predicates` is an advanced use of incremental models, where data volume is large enough to justify additional investments in performance. This config accepts any valid SQL expression, however dbt does not check the syntax. 
+
+For example, this is a pattern we might expect to see on Snowflake:
+
+```yml
+models:
+  - name: my_incremental_model
+    config:
+      materialized: incremental
+      unique_key: id
+      # this will affect how the data is stored on disk, and indexed to limit scans
+      cluster_by: ['session_start']  
+      incremental_strategy: merge
+      # this limits the scan of the existing table to the last 7 days of data
+      incremental_predicates: "DBT_INTERNAL_DEST.session_start > datediff(day, -7, current_date)"
+      # `DBT_INTERNAL_DEST` and `DBT_INTERNAL_SOURCE` are the standard aliases for the target table and temporary table, respectively, during an incremental run using the merge strategy. 
+```
+
+This will template (in the `dbt.log` file) a `merge` statement like:
+```sql
+merge into <existing_table> DBT_INTERNAL_DEST
+    from <temp_table_with_new_records> DBT_INTERNAL_SOURCE
+    on
+        -- unique key
+        DBT_INTERNAL_DEST.id = DBT_INTERNAL_SOURCE.id
+        and
+        -- custom predicate: limits data scan in the "old" data / existing table
+        DBT_INTERNAL_DEST.session_start > datediff(day, -7, current_date)
+    when matched then update ...
+    when not matched then insert ...
+```
+
+You will still want to limit the data scan of _upstream_ tables within the body of their incremental model SQL. This is to limit the amount of "new" data that's processed/transformed
```

Comment:
@runleonarun, how does it read like this (just removing ambiguity in the directions):
```suggestion
Limit the data scan of _upstream_ tables within the body of their incremental model SQL, which will limit the amount of "new" data processed/transformed.
```

Reply:
@matthewshaver much clearer!